cms/djangoapps/contentstore/views/course.py

```diff
@@ -1220,6 +1223,37 @@ def advanced_settings_handler(request, course_key_string):
                         # now update mongo
                         modulestore().update_item(course_module, request.user.id)
 
+                        """
```

Comment:
Please note that I have added the changes to create a model instance in this view because course modules are not straightforward mongo documents where I can add these changes in a save or on post save function. So I believe it will be a lot easier to maintain and debug if the edx changes remain here.